src/sdk/Recognizer.ts

```diff
@@ -87,6 +93,15 @@ export abstract class Recognizer {
         this.dispose(true);
     }
 
+    /**
+     * @Internal
+     * Internal data member to support fromRecognizer* pattern methods on other classes.
+     * Do not use externally, object returned will change without warning or notive.
+     */
+    public get internalData(): object {
```

Comment:
> public get internalData(): [](start = 3, length = 27)

can we hide this here or otherwise this will end up in the public api for the recognizer?

Reply:
Given our mixed pattern of factory and constructor I didn't find a way to support all the FromRecognizer* methods in plan and not expose some data that was public that wasn't more invasive.